lib/schema_plus/core/sql_struct.rb

```diff
@@ -12,6 +16,7 @@ def parse!(sql)
             \( \s*
             (?<body>.*) \s*
             \) \s*
+            #{INHERITANCE_REGEX if sql[INHERITANCE_KEY]}
```

Comment:
This would be simpler using an optional term in the regex:

```
(?<inheritance> INHERITS \s* \( .( \) )? \s*
```
  I don't think all those constant definitions are needed.